test.sh

```diff
@@ -16,6 +16,32 @@ function main(){
     else
         run_production_tests;
     fi
+
+    # TODO: currently only work on development settings
```

Comment:
If you state what's broken/what the current unwanted limitation is, use `FIXME` rather than `TODO`. If you prefer `TODO`, state what should be changed, not what the current state is.